### PR TITLE
Update the What's new next template

### PR DESCRIPTION
**Pull request by 09jvilla**
Call out that you might want to add an image or a video to your 'what's new next' callout.

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Or if all three:
 ## _`DESCRIPTION`_
 
 Include an overview of the feature and problem it solves, and where to learn more.
-Link to any appropriate documentation.
+Link to any appropriate documentation and optionally embed a video or image to illustrate the feature.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/README.md:58]:
```suggestion
Link to any appropriate documentation and, optionally, embed a video or image to illustrate the feature. 
```

**Review by imatwawana — APPROVED** [review]

**Comment by 09jvilla** [comment]
@imatwawana - I added a link in there. Can you re-review for me?

File: docs/sources/whatsnew/whats-new-next/README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Or if all three:
 ## _`DESCRIPTION`_
 
 Include an overview of the feature and problem it solves, and where to learn more.
-Link to any appropriate documentation.
+Link to any appropriate documentation and, optionally, embed a video or image to illustrate the feature, following our [image and media guidlines](https://grafana.com/docs/writers-toolkit/write/image-guidelines/#image-diagram-and-screenshot-guidelines).
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/README.md:58]: Fixed a typo 
```suggestion
Link to any appropriate documentation and, optionally, embed a video or image to illustrate the feature, following our [image and media guidelines](https://grafana.com/docs/writers-toolkit/write/image-guidelines/#image-diagram-and-screenshot-guidelines). 
```

**Review by imatwawana — APPROVED** [review]
Thanks for the update!

**Comment by 09jvilla** [comment]
@imatwawana Can you merge on my behalf? I think we're ready to go.

**Pull request merged**